src/parse.ts

```diff
@@ -12,11 +12,13 @@ import { Grammar, Parser } from "nearley";
 
 import { buildRos2Type } from "./buildRos2Type";
 import ros1Rules from "./ros1.ne";
+import ros2idlRules from "./ros2idl.ne";
 
 const ROS1_GRAMMAR = Grammar.fromCompiled(ros1Rules);
+const ROS2IDL_GRAMMAR = Grammar.fromCompiled(ros2idlRules);
 
 export type ParseOptions = {
-  /** Parse message definitions as ROS2. Otherwise, parse as ROS1 */
+  /** Parse message definitions as ROS 2. Otherwise, parse as ROS1 */
```

Comment:
OSRF will be very happy

Reply:
`ROS1` officially outside of OSRF sphere of opinion